packages/jsii-calc/package.json

```diff
@@ -37,18 +37,16 @@
     "watch": "jsii --project-references -w",
     "lint": "eslint . --ext .js,.ts --ignore-path=.gitignore",
     "lint:fix": "yarn lint --fix",
-    "test": "node test/test.calc.js && diff-test test/assembly.jsii .jsii",
+    "test": "(ls test/test.*.js | xargs -n1 node) && diff-test test/assembly.jsii .jsii",
```

Comment:
Can we not just use `jest` like the grown ups do? This `ls | xargs` contraption is not very Windows-friendly.

Reply:
Do we have a goal to be able to build jsii on Windows?

Reply:
Deferring to a future commit